app/policies/neighbourhood_policy.rb

```diff
@@ -32,7 +32,7 @@ def set_users?
 
   def permitted_attributes
     if user.root?
-      %i[name name_abbr unit unit_code_key unit_code_value unit_name].push(user_ids: [])
+      %i[name name_abbr unit unit_code_key unit_code_value unit_name release_date].push(user_ids: [])
```

Comment:
I agree with you that this is a bug and think this is definitely something we should fix but is out of the scope of this problem. I've added the `release_date` into this list because it was decided at some point that roots should be able to edit all neighbourhood fields, and if that is the case, they need to also be able to edit the release date. (wondered if this was so they could add new non geographical neighbourhoods, for example?)

Reply:
sounds like one of those choices that has bad points on either side of the decision so I am just going to okay this as it isn't causing and runtime problems and we can fix this later on.